src/net_processing.h

```diff
@@ -90,4 +90,7 @@ struct CNodeStateStats {
 /** Get statistics from node state */
 bool GetNodeStateStats(NodeId nodeid, CNodeStateStats &stats);
 
+/** Relay transaction to every node */
+void RelayTransaction(const uint256&, const CConnman& connman);
```

Comment:
nit: Please add the name of the txid argument:

`void RelayTransaction(const uint256& txid, const CConnman& connman);`